### PR TITLE
Verify LoginPacket

### PR DESCRIPTION
**Pull request by johnnywoof**
Added a way so that the signatures from LoginPackets can be verified. More specifically, it checks the signature for the chain JWT token that contains the player's uuid and display name. This will allow the proxy to block login requests that were not authenticated with microsoft/mojang.
This essentially makes it so that the proxy can accept connections in "online mode"

~~The only problem is that the public key will need to be manually updated if they decide to change it. Although, it is simple as replacing the contents in the immediate ca public key file.~~
I made another commit that uses the root public key from mojang. I'm not sure when it expires, but it remained the same for well over two years.

### CODE REVIEW

**Comment by johnnywoof** [comment]
The only problem is that the public key will need to be manually updated if they decide to change it. Although, it is simple as replacing the contents in the immediate ca public key file.

**Comment by DefinitlyEvil** [comment]
ah hah, with this verification, we can improve CLSv2 a lot! let's see what the team members say.

**Review by DefinitlyEvil — APPROVED** [review]
looks legit and useful

File: src/main/resources/en_US.properties [hunk 1]
```diff
@@ -44,4 +44,5 @@ message_cls_notice=Dragonet CLS system enabled. \n\u00a7eYou can login at \u00a7
 message_server_error=\u00a7cServer was unable to log you in because it had an error! \n\u00a7cPlease try join again later... :( \n\u00a70.\n\u00a7cReason: %s
 error_cls_unreachable=Unable to reach CLS HTTP API
 error_cls_error=CLS System replied weird stuff. Please retry.
-error_cls_login=Faild to login with your login token, try re-login on CLS? 
+error_cls_login=Faild to login with your login token, try re-login on CLS?
```
Comment (ApocalypsjeNL) [inline src/main/resources/en_US.properties:47]: Isn't it Failed instead of Faild?

Comment (lukeeey) [reply src/main/resources/en_US.properties:47]: @DefinitelyEvil has always said faild, idk why

Comment (DefinitlyEvil) [reply src/main/resources/en_US.properties:47]: meh that's my bad... xD (so embarrasing)

**Pull request merged**